node/res/bp.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Never deployed non-raw specs before and I am not sure if they will work. Probably will?

Reply:
Yep, they do